Cargo.toml

```diff
@@ -135,7 +135,11 @@ hwloc-2_8_0 = ["hwlocality-sys/hwloc-2_8_0", "hwloc-2_5_0"]
 # - Make it enable features associated with all previous versions.
 # - Change hwloc-latest to point to that feature.
 # - Update hwlocality-sys' Cargo.toml similarly
-# - Update hwlocality-sys' build.rs to up the build requirements.
+# - Update hwlocality-sys' build.rs to...
+#   * Record the new feature's hwloc version requirements in `setup_hwloc()`
+#   * Use a newer official tarball, with appropriate checksum (can be computed
+#     using `curl <tar.gz url> | sha3-256sum`), in `setup_vendored_hwloc()`
```

Comment:
Interesting choice of Sha3. There are already hashsums published on the official website (MD5, SHA1 and SHA256). This step can be replaced with looking up the hash on the website if more common SHA256 is used.